composer/core/state.py

```diff
@@ -823,6 +823,7 @@ def _get_state_metadata(self) -> Dict[str, Any]:
         """
         metadata_dict = {}
         metadata_dict['composer_env_info'] = get_composer_env_dict()
+        metadata_dict['torch_version'] = torch.__version__
```

Comment:
Not sure if this is the best interface, especially if we're considering exposing other package versions in the future